hw05.Rmd

```diff
@@ -0,0 +1,114 @@
+---
+title: "hw05"
+author: "Maria Evers"
+date: "8/3/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+
+##Homework 5 part 1:
+```{r}
+
+library(tidyverse)
```

Comment:
Since you're loading tidyverse here, I think it already includes all of the other packages you've loaded except for knitr.